tests/Integration/Behaviour/Features/Scenario/CustomerService/customer_service.feature

```diff
@@ -0,0 +1,23 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s customer_service --tags customer-management
+@restore-all-tables-before-feature
+@customer-management
+Feature: Customer service
+
+  Scenario: Add customer thread
+    When I add new customer thread "thread1" with following properties:
+      | message   | test message |
+      | contactId | 2            |
```

Comment:
We avoid relying on direct ID in the behat tests, you can instead create a new one for your purpose and assign its ID to a reusable reference (`AddContactCommand` already has an available behat step you can rely on)